src/main/java/xyz/kmahyyg/eshopdemo/controller/OrderController.java

```diff
@@ -1,29 +1,102 @@
 package xyz.kmahyyg.eshopdemo.controller;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.configurationprocessor.json.JSONArray;
+import org.springframework.boot.configurationprocessor.json.JSONObject;
+import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import xyz.kmahyyg.eshopdemo.dao.SysItemsDao;
 import xyz.kmahyyg.eshopdemo.dao.SysOrdersDao;
+import xyz.kmahyyg.eshopdemo.dao.SysUsersDao;
+import xyz.kmahyyg.eshopdemo.enums.OrderStatusEnum;
+import xyz.kmahyyg.eshopdemo.model.SysItems;
 import xyz.kmahyyg.eshopdemo.model.SysOrders;
-import xyz.kmahyyg.eshopdemo.utils.UserInfoUtil;
+import xyz.kmahyyg.eshopdemo.security.UserInfo;
 
```

Comment:
Please do not directly use `UserInfo` for acquring UID, since I've already made this much more simple `UserInfoUtil` for you.